docs/content/getting-started/perform-check.mdx

```diff
@@ -1,6 +1,7 @@
 ---
 title: Perform a Check
 sidebar_position: 4
+toc_max_heading_level: 4
```

Comment:
to maintain proper hierarchy and also show this option in the table of contents:
<img width="332" alt="image" src="https://github.com/user-attachments/assets/cf0399af-7c9d-4c3b-833a-b09db5ab5e34">